### PR TITLE
feature: S3C-2035: KMIP, TLS files from env

### PR DESCRIPTION
**Pull request by ploki**
Load the content of the TLS certificates and key when the configuration is provided through the environment variables.

### CODE REVIEW

**Review by alexanderchan-scality — APPROVED** [review]

File: lib/Config.js [hunk 1]
```diff
@@ -814,11 +835,12 @@ class Config extends EventEmitter {
                      * must become 'hosts'
                      */
                     host: process.env.S3KMIP_HOSTS,
-                    key: process.env.S3KMIP_KEY || undefined,
-                    cert: process.env.S3KMIP_CERT || undefined,
-                    ca: process.env.S3KMIP_CA
-                        ? process.env.S3KMIP_CA.split(',')
-                        : undefined,
+                    key: this._loadTlsFile(process.env.S3KMIP_KEY || undefined),
```
Comment (alexanderchan-scality) [inline lib/Config.js:838]: ~~should `_loadTlsFile` be called here? I may be wrong, but it seems like loading the file here will cause the call on https://github.com/scality/cloudserver/pull/1759/files#diff-9280903abc69f09eff9ab36070b6d751R888 to throw an error~~ confused this.kmip with config.kmip

**Review by tmacro — APPROVED** [review]

File: lib/Config.js (hunk 1 above)
Comment (ploki) [reply lib/Config.js:838]: I don't see why it would throw an error, can you be more specific?

Comment (ploki) [reply lib/Config.js:838]: does this comment apply to the other members of the tls struct? (e.g. ca and cert)

Comment (alexanderchan-scality) [reply lib/Config.js:838]: you are correct. upon double checking, there won't be an error. I have mixed up `this.kmip` vs `config.kmip`

Comment (ploki) [reply lib/Config.js:838]: I did too, the reason for this PR...

**Review by dora-korpar — APPROVED** [review]

**Comment by ploki** [comment]
@bert-e approve

**Comment by ploki** [comment]
@bert-e reset

**Pull request merged**